uber/templates/signups/onsite_jobs.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "base-admin.html" %}
 {% block title %}On-Site Shifts{% endblock %}
 {% block backlink %}{% endblock %}
```

Comment:
btw mostly unrelated to this PR:, mental note, anytime you see ```{% block backlink %}{% endblock %}``` you should mentally translate that as 'don't display the main menu'.   In the future we should probably retire the name 'backlink'